### ISSUE TITLE
Update current detraction percentage based on the latest revision of the law

### ISSUE THREAD

**Issue by davide-aloi** [andybergon/rientro-cervelli#41]
I believe that with the latest revision of the regulation the amount of tax relief has been decreased from 70% to 50%. It would be beneficial to update this, or let the user define the percentage.

(amazing work btw)

**Comment by hipoglucido** [comment]
Indeed, since 2025 there are new changes and it would be great to be able to keep using tis website! Not only  it changed to 50% in general, but also with the option to be 60% instead if you have a kid with you.

So I agree letting user define the percentage with be a great addition
